File: q4.java
// You are given an array find sum of elts of array
public class q4 {
    public static void main(String[] args) {
        int [] ar = {2, 4, 8, 8,8,28};
        int sum = 0;

        for(int i = 0; i < ar.length; i++){
            sum += ar[i];
        }

        System.out.println(sum);
    }
}

File: q4.cpp
// You are given an array find sum of elts of array
#include<bits/stdc++.h>
using namespace std;

int main() {
    int ar[] = {4,5,68,9,29,83,92,9,1};
    int sum = 0;

    for(int i = 0; i < 9; i++){
        sum += ar[i];
    }

    cout << sum;
    return 0;
}